build-tools/bin/run_tests.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
-set -x
 set -o nounset
```

Comment:
I prefer to have set -x so we don't need to manually change the script and rerun to debug a failure and find out what caused it.